src/JWT.Extensions.AspNetCore/JWT.Extensions.AspNetCore.csproj

```diff
@@ -15,6 +15,8 @@
     <AssemblyVersion>6.0.0.0</AssemblyVersion>
     <RootNamespace>JWT</RootNamespace>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <SignAssembly>true</SignAssembly>
```

Comment:
Can this be centralized using Directory.Build.props? See https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019#directorybuildprops-and-directorybuildtargets
You can drop it /src or the root (see my another comment)